### ISSUE TITLE
Add website accessibility link to footer 

### ISSUE THREAD

**Issue by selina-aragon** [edcarp/edcarp.github.io#8]
As part of the guidelines, the website accessibility statement should be linked from a place where it can be accessed at all times (e.g. the footer). @gperu could you move the statement from the bottom of the Edinburgh Carpentries landing page to the footer to comply with regulations?
